R/plot_jags_rhatdx.R

```diff
@@ -61,19 +61,25 @@ plot_jags_Rhat <- function(data,  # nolint: object_name_linter
 
       visualize_jags_plot <- visualize_jags_plot |>
         # Changing parameter name to reflect the input
-        dplyr::mutate(Parameter = paste0("iso = ", j, ", parameter = ",
-                                         .data$Parameter_sub, ", strat = ",
-                                         i),
+        dplyr::mutate(Parameter = paste0(.data$Parameter_sub),
```

Comment:
this line seems like it can be simplified, or possibly eliminated?

```suggestion
        dplyr::mutate(Parameter = .data$Parameter_sub,
```

Reply:
> this line seems like it can be simplified, or possibly eliminated?

@sschildhauer up to you if you want to try to change this or not; otherwise, you should be able to merge as-is (after resolving the merge conflict